### ISSUE TITLE
Better npc shopping

### ISSUE THREAD

**Issue by kalilistic** [goatcorp/suggestions#547]
### Description
Add search and sorting to vendors.

### More info
https://discord.com/channels/581875019861328007/685275026156683280/872503547898519552

----

 - [x] I checked if this request wasn't already proposed
 - [x] I made sure this request follows Dalamud's [plugin guidelines](https://github.com/goatcorp/FFXIVQuickLauncher#isnt-this-cheating)

**Comment by kalilistic** [comment]
Condensed list requested: https://discord.com/channels/581875019861328007/685275026156683280/939965756173606983

**Comment by cohenaj194** [comment]
Yes I would love this as well. I really want to be able to sort vendor item lists by alphabetical order. 

https://discord.com/channels/581875019861328007/685275026156683280/1043556410065170472
